pkg/webhooks/mutator/hyperConvergedMutator.go

```diff
@@ -110,6 +110,25 @@ func (hcm *HyperConvergedMutator) mutateHyperConverged(_ context.Context, req ad
 
 	}
 
+	if hc.Spec.MediatedDevicesConfiguration != nil {
+		if len(hc.Spec.MediatedDevicesConfiguration.MediatedDevicesTypes) > 0 && len(hc.Spec.MediatedDevicesConfiguration.MediatedDeviceTypes) == 0 { //nolint SA1019
+			patches = append(patches, jsonpatch.JsonPatchOperation{
+				Operation: "add",
+				Path:      "/spec/mediatedDevicesConfiguration/mediatedDeviceTypes",
+				Value:     hc.Spec.MediatedDevicesConfiguration.MediatedDevicesTypes, //nolint SA1019
+			})
+		}
+		for i, hcoNodeMdevTypeConf := range hc.Spec.MediatedDevicesConfiguration.NodeMediatedDeviceTypes {
+			if len(hcoNodeMdevTypeConf.MediatedDevicesTypes) > 0 && len(hcoNodeMdevTypeConf.MediatedDeviceTypes) == 0 { //nolint SA1019
+				patches = append(patches, jsonpatch.JsonPatchOperation{
+					Operation: "add",
+					Path:      fmt.Sprintf("/spec/mediatedDevicesConfiguration/nodeMediatedDeviceTypes/%d/mediatedDeviceTypes", i),
+					Value:     hcoNodeMdevTypeConf.MediatedDevicesTypes, //nolint SA1019
+				})
+			}
+		}
```

Comment:
Do we also want to clear to MediatedDevicesTypes fields?

Reply:
No, I think we should not: they are deprecated but still there and some external code could also try to read it and we don't want to break it.